tests/v2/validation/rbac/rbac_test.go

```diff
@@ -0,0 +1,249 @@
+		//Length of namespace list for admin and cluster owner should match
+		assert.Equal(rb.T(), len(namespaceListAdmin), len(namespaceListClusterMembers))
+		//Namespaces obtained as admin and cluster owner should be same
+		assert.Equal(rb.T(), namespaceListAdmin, namespaceListClusterMembers)
+	case roleMember:
+		require.Error(rb.T(), err)
+		errMessage := strings.Split(err.Error(), ":")[0]
+		//assert cluster member is not able to list namespaces
+		assert.Equal(rb.T(), "namespaces is forbidden", errMessage)
+	}
+
+	//Testcase6 Validate if cluster members are able to delete the namespace in the project they are not owner of
+	log.Info("Testcase6 - Validating if ", role, " can delete a namespace from a project they are not owner of.")
+	err = namespaces.DeleteNamespace(rb.standardUserClient, namespaceName, rb.cluster.ID)
+	switch role {
+	case roleOwner:
+		require.NoError(rb.T(), err)
+	case roleMember:
+		require.Error(rb.T(), err)
+	}
+}
+
+func (rb *rbTestSuite) ValidateDeleteProject(role string) {
+
+	//Testcase7 Validate if cluster members are able to delete the project they are not owner of
+	err := rb.standardUserClient.Management.Project.Delete(rb.adminProject)
+
+	switch role {
+	case roleOwner:
+		require.NoError(rb.T(), err)
+	case roleMember:
+		require.Error(rb.T(), err)
+		errStatus := strings.Split(err.Error(), ".")[1]
+		rgx := regexp.MustCompile(`\[(.*?)\]`)
+		errorMsg := rgx.FindStringSubmatch(errStatus)
+		assert.Equal(rb.T(), "403 Forbidden", errorMsg[1])
+	}
+}
+
+func (rb *rbTestSuite) ValidateRemoveClusterRoles(role string) {
+
+	//Testcase8 Remove added cluster member from the cluster as an admin
+	err := users.RemoveClusterRoleFromUser(rb.client, rb.standardUser)
+	require.NoError(rb.T(), err)
+
+}
+
+func (rb *rbTestSuite) TestRBAC() {
+	tests := []struct {
+		name        string
+		clusterRole string
+	}{
+		{"Cluster Owner", "cluster-owner"},
+		{"Cluster Member", "cluster-member"},
+	}
+	for _, tt := range tests {
+		rb.Run(tt.name, func() {
+			newUser, err := createUser(rb.client)
+			require.NoError(rb.T(), err)
+			rb.standardUser = newUser
+			rb.T().Logf("Created user: %v", rb.standardUser.Username)
+			rb.standardUserClient, err = rb.client.AsUser(newUser)
+			require.NoError(rb.T(), err)
+			
+			subSession := rb.session.NewSession()
+			defer subSession.Cleanup()
+
+			createProjectAsAdmin, err := createProject(rb.client, rb.cluster.ID)
+			rb.adminProject = createProjectAsAdmin
+			require.NoError(rb.T(), err)
+		})
+
+		//Verify standard users cannot list any clusters
+		rb.Run(tt.name, func() {
+			rb.T().Logf("Test case Validate standard users cannot list any downstream clusters")
+			_, err := listClusters(rb.standardUserClient)
+			require.Error(rb.T(), err)
+			assert.Equal(rb.T(), "Resource type [provisioning.cattle.io.cluster] is not listable", err.Error())
+		})
+
+		rb.Run(tt.name, func() {
+			//Adding created user to the downstream clusters with the specified roles.
```

Comment:
@igomez06 Can we not wait for the PR https://github.com/rancher/rancher/pull/39341 to be merged in and merge this PR if everything looks ok ? Since we want a few test cases for RBAc to be addressed in v2.7.0 and not have them in this PR.

Reply:
Well Anu, PR https://github.com/rancher/rancher/pull/39341 is vital for v1 calls, and delaying more not only adds more work for me, but it'll keep pushing this off, which is super critical to our testing. You can branch off of this branch and add tests.